Snakefile

```diff
@@ -1303,6 +1322,9 @@ rule tmhmm_parallel:
 
     # open the input file, output file and log file
     with open(input[0], "r") as input_handle, open(output[0], "a+") as output_handle, open(log[0], "a+") as log_handle:
+      # print version to log
+      version = os.popen('conda list | grep tmhmm-py').read()
```

Comment:
I think this should work in python: `import tmhmm; print(tmhmm.__version__)`

Reply:
It should, but it does not for this package... `AttributeError: module 'tmhmm' has no attribute '__version__'`

Reply:
How about this?
```
import pip
pip.main(['show', 'tmhmm.py'])
```

Reply:
This works however it gives a following warning 
```
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
```

So I can change it to `subprocess.run(['pip','show','tmhmm.py'],stdout=log_handle,stderr=log_handle)`